centreon/www/include/common/listing/listing.css

```
@@ -0,0 +1,1365 @@
+    left: 12px;
+    display: flex;
+    color: var(--c-text-secondary, #6b7280);
+    pointer-events: none;
+}
+
+.cl-search .cl-search-icon svg {
+    width: 16px;
+    height: 16px;
+    /* Override the theme's global svg { fill } so the magnifier stays outlined */
+    fill: none !important;
+    stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)